ChargeRateMonitoring.c

```diff
@@ -0,0 +1,21 @@
+#include "Config/ParameterConfig.h"
+#include "ChargeRateMonitoring.h"
+#include "MathUtil.h"
+
+
+ChargeRateControlData_t ChargeRateControlData = {
+    BMS_CHARGERAGE_LOWER_RANGE,
+    Failuretype_NoFailure,
+    "ChargeRate"
+     };
+
+int IsBatterychargeRateStable(float chargeRate)
+{
+  if(IsValueOutofHigherRangeThreshold(chargeRate,ChargeRateControlData.HigherOutOfRangeThreshold))
+  {
+    ChargeRateControlData.Failure = Failuretype_OutOfHigherRange;
+    DisplayFailureOnConsole(ChargeRateControlData.Identifier,
```

Comment:
`DisplayFailureOnConsole` is in 7 places. If we need to do something else than console, we need to change this.
Worse, if some clients want console, some others want a text-to-speech, we need to test in 7 places.
Any other way?